qiskit/dagcircuit/dagcircuit.py

```diff
@@ -93,6 +93,9 @@ def __call__(self):
 
         self._global_phase = 0
 
+        self.duration = None
+        self.instruction_durations = None
```

Comment:
This was removed from most places, is it left here on purpose?

Reply:
No. Good catch! Thank you.